omni-relayer/src/utils/near.rs

```diff
@@ -88,10 +88,51 @@ pub async fn handle_streamer_message(
                 )
                 .await;
             }
-            Nep141LockerEvent::FinTransferEvent { .. }
-            | Nep141LockerEvent::LogMetadataEvent { .. }
-            | Nep141LockerEvent::SignClaimNativeFeeEvent { .. }
-            | Nep141LockerEvent::ClaimFeeEvent { .. } => {}
+            Nep141LockerEvent::FinTransferEvent {
+                ref nonce,
+                ref transfer_message,
+            } => {
+                if nonce.is_none() {
+                    utils::redis::add_event(
+                        redis_connection,
+                        utils::redis::NEAR_FIN_TRANSFER_EVENTS,
+                        transfer_message.origin_nonce.0.to_string(),
+                        log,
+                    )
+                    .await;
+                }
+            }
+            Nep141LockerEvent::ClaimFeeEvent {
+                ref transfer_message,
+                ref native_fee_recipient,
+            } => {
+                if native_fee_recipient == &config.near.relayer_address_on_evm {
+                    utils::redis::add_event(
+                        redis_connection,
+                        utils::redis::NEAR_FIN_TRANSFER_EVENTS,
```

Comment:
I don't like that in the case of a ClaimFeeEvent, you're saving the event in NEAR_FIN_TRANSFER_EVENT. This is not accurate. I understand that it's done because the further processing is the same, but in that case, the column should be named differently.

For example, SIGN_CLAIM_NATIVE_FEE_QUEUE

Reply:
Yep, it makes sense 
Fixed in eb3e8a5d772bfb61d88b550f239fcefe8dd560a5